### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by naitik0009** [community/community#69792] [GitHub Education]
### Hi everyone,

I'm unable to register for github student developer pack it says There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact?tags=education-support) for more information.

**Comment by mantasu** [comment]
Yeah, on [GitHub Support](https://support.github.com/contact?tags=education-support) just click "**GitHub Education program or benefits**" > "**Contact us**" and just enter your details. It will raise the ticket and the staff will get back to you. That's what I did when I had issues with student approval.

**Reply by yesbhautik** [reply]
tried many times, not getting responses from github team, very unprofessional
